packages/babel-plugin-open-source/README.md

```diff
@@ -23,11 +23,25 @@ Add the plugin to your babel config.
 // .babelrc
 {
    "plugins": [
-      "babel-plugin-open-source"
+      "babel-plugin-open-source",
+      "editor:"vscode", // vscode-insiders | sublime | atom
     ]
 }
 ```
 
+Current Supoorted editors are:
+- VSCode
+- VSCode-Insiders
+- Sublime
+- Atom
+- Phpstorm
+
+Personalized editors
```

Comment:
Can we rephrase this as - "If folks in your team use different editors, you can customise the editor just for you by adding ..."

Reply:
Pushed the changes.